python-threatexchange/threatexchange/tests/hashing/test_pdq_hash_rotations.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+from unittest.mock import patch, MagicMock
+import pathlib
+
+from threatexchange.cli.hash_cmd import HashCommand
+from threatexchange.cli.cli_config import CLISettings
+from threatexchange.cli.exceptions import CommandError
+from threatexchange.content_type.photo import PhotoContent, RotationType
+from threatexchange.content_type.text import TextContent
+from threatexchange.content_type.url import URLContent
+from threatexchange.content_type.video import VideoContent
+
+
+class TestHashCommand(unittest.TestCase):
```

Comment:
blocking: We are using the py.test framework for this library, not unittest. 

Check out: https://docs.pytest.org/en/stable/

and this reference test: https://github.com/haianhng31/ThreatExchange/blob/8d10599cda78afa49f763c08ca556db94f4ae332/python-threatexchange/threatexchange/cli/tests/match_cmd_test.py

We should probably just use the hash and match_cmd tests rather than creating new tests.